### ISSUE TITLE
(March 2023) Extension removed from Mozilla website

### ISSUE THREAD

**Issue by NeatNit** [vladikoff/netflix-1080p-firefox#63]
The extension is no longer available, following [the link from the readme](https://addons.mozilla.org/en-US/firefox/addon/force-1080p-netflix/) says:

> If you’ve followed a link from another site for an extension or theme, that item is no longer available.

I'm not sure when exactly it was removed, but it couldn't have been that long ago.

Related: #28

**Comment by NeatNit** [comment]
Since I already had this extension in a different profile, here is the signed extension:

[{f18f0257-10ad-4ff7-b51e-6895edeccfc8}.zip](https://github.com/vladikoff/netflix-1080p-firefox/files/10981084/f18f0257-10ad-4ff7-b51e-6895edeccfc8.zip)

Rename from .zip to .xpi and install via about:addons:

![image](https://user-images.githubusercontent.com/2400026/225348180-80800754-32fd-4500-a41e-de179f5ce03d.png)

**Comment by Parap0nera** [comment]
> 

Thank you so much!

**Comment by DraconicNEO** [comment]
@vladikoff Would it be possible to just release the xpi of the extension here on GitHub without relying on the AMO repository? That way people could still use your extension even if Mozilla's Addon store rejects it since Firefox still allows addons from files and third party extension websites (can install directly from github).

**Comment by ciberponk** [comment]
Thank you very much ^_^

**Comment by gelie** [comment]
> Since I already had this extension in a different profile, here is the signed extension:
> 
> [{f18f0257-10ad-4ff7-b51e-6895edeccfc8}.zip](https://github.com/vladikoff/netflix-1080p-firefox/files/10981084/f18f0257-10ad-4ff7-b51e-6895edeccfc8.zip)
> 
> Rename from .zip to .xpi and install via about:addons:
> 
> ![image](https://user-images.githubusercontent.com/2400026/225348180-80800754-32fd-4500-a41e-de179f5ce03d.png)

**Comment by gelie** [comment]
Thank you @NeatNit . Anyone know why it was removed?

**Comment by adamency** [comment]
@vladikoff Can you please acknowledge this thread and let us know the reason why the extension is not available anymore on Mozilla store ? Even if you don't have any more time to spend in this project, this would help the community determine how to go from there. Thanks in advance.

**Comment by vladikoff** [comment]
Hi @adamency , There were multiple reasons why it got removed, main one was that the cadmium player is a minified blob and they don't like that. https://github.com/vladikoff/netflix-1080p-firefox/blob/master/src/cadmium-playercore-1080p.js

**Comment by DraconicNEO** [comment]
> Hi @adamency , There were multiple reasons why it got removed, main one was that the cadmium player is a minified blob and they don't like that. https://github.com/vladikoff/netflix-1080p-firefox/blob/master/src/cadmium-playercore-1080p.js

If you can't host it on the add-ons store would it be possible to continue hosting it here on GitHub? I mean Firefox lets people install add-ons from GitHub that aren't in the repository.

**Comment by criminalvibe** [comment]
> Since I already had this extension in a different profile, here is the signed extension:
> 
> [{f18f0257-10ad-4ff7-b51e-6895edeccfc8}.zip](https://github.com/vladikoff/netflix-1080p-firefox/files/10981084/f18f0257-10ad-4ff7-b51e-6895edeccfc8.zip)
> 
> Rename from .zip to .xpi and install via about:addons:
> 
> ![image](https://user-images.githubusercontent.com/2400026/225348180-80800754-32fd-4500-a41e-de179f5ce03d.png)


Downloaded it from your link, yet it's still in 720p. Any help?

**Comment by comfortablemilk4454** [comment]
@criminalvibe what platform are you on? and try using [this xpi file](https://cdn.discordapp.com/attachments/1177337848416972886/1181609932966068304/f18f0257-10ad-4ff7-b51e-6895edeccfc8.xpi?ex=6581af01&is=656f3a01&hm=b0795230776c65ffff4d5dbd1d5e0deb08bdb2c864118d3bae4ebccf0b1b13ec&) instead of renaming the zip to xpi in case you're doing that wrong, which most ppl don't know that you have to change an additional setting in your windows/macOS to be able to change the file type, not just by putting an .xpi at the end and removing the .zip from the title without first changing said setting.

**Comment by criminalvibe** [comment]
> @criminalvibe what platform are you on? and try using [this xpi file](https://cdn.discordapp.com/attachments/1177337848416972886/1181609932966068304/f18f0257-10ad-4ff7-b51e-6895edeccfc8.xpi?ex=6581af01&is=656f3a01&hm=b0795230776c65ffff4d5dbd1d5e0deb08bdb2c864118d3bae4ebccf0b1b13ec&) instead of renaming the zip to xpi in case you're doing that wrong, which most ppl don't know that you have to change an additional setting in your windows/macOS to be able to change the file type, not just by putting an .xpi at the end and removing the .zip from the title without first changing said setting.

I'm on Windows 11, using Firefox.

**Comment by criminalvibe** [comment]
Also, got it to work with your link. Thank you very much. However, it still looks really bad lol. Might just have to sail the high seas.

**Comment by comfortablemilk4454** [comment]
Perhaps it could be your internet connection?
________________________________

**Comment by irort** [comment]
> @criminalvibe what platform are you on? and try using [this xpi file](https://cdn.discordapp.com/attachments/1177337848416972886/1181609932966068304/f18f0257-10ad-4ff7-b51e-6895edeccfc8.xpi?ex=6581af01&is=656f3a01&hm=b0795230776c65ffff4d5dbd1d5e0deb08bdb2c864118d3bae4ebccf0b1b13ec&) instead of renaming the zip to xpi in case you're doing that wrong, which most ppl don't know that you have to change an additional setting in your windows/macOS to be able to change the file type, not just by putting an .xpi at the end and removing the .zip from the title without first changing said setting.

Unfortunately, this extension has recently become invalid. When I play Netflix episodes on Firefox, the UI3013 error is displayed. The official help center does not have a solution to this error. After disabling the extension, Netflix plays normally.

**Comment by comfortablemilk4454** [comment]
you're right. i get that error too with the extension, and without it netflix plays perfectly fine. oh well. its about time to switch to safari on my apple silicon mac for watching netflix (fuck safari for everything else) as with apple silicon & safari netflix lets the quality go up to 4k over there.
